package.json

```diff
@@ -36,6 +36,7 @@
     "sinon": "1.17.6"
   },
   "peerDependencies": {
+    "flow-bin": ">= 0.30.0 < 0.40.0",
```

Comment:
Wouldn't this be the same as `^0.30.0`? I find the docs a little confusing though: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004 Maybe it's better to make it explicit? :)